aio/content/guide/upgrade-lazy.md

```diff
@@ -1,4 +1,4 @@
-# Upgrading from AngularJS (in a more flexible way)
+# Migrating from AngularJS to Angular
```

Comment:
The tough part is that we have two upgrade guides, which talk about the same thing (upgrading/migrating from AngularJS to Angular). The difference is in whether you use `UpgradeModule` or `downgradeModule()`. Both do the same thing (allow creating a hybrid app) and the differences are quite low-level (so not a good fit for a title).

Maybe we could incorporate this guide into the other one?

Reply:
As per an offline conversation, @gkalpak and I decided to keep it separate for the time being.